Pester.psm1

```diff
@@ -659,6 +659,40 @@ function Get-ScriptBlockScope
     [scriptblock].GetProperty('SessionStateInternal', $flags).GetValue($ScriptBlock, $null)
 }
 
+function Get-OperatingSystem
+{
+    [CmdletBinding()]
+    param()
+
+    if ($PSVersionTable.PSVersion.Major -lt 6)
+    {
+        'Windows'
+    } 
+    elseif ($IsOSX)
+    {
+        'OSX'
+    }
+    elseif ($IsLinux)
+    {
+        'Linux'
+    }
+}
+
+function Get-TempDiretory
```

Comment:
Typo here, Diretory instead of Directory.  That's the cause of most of the failing tests.